custom_components/evohome_cc/sensor.py

```diff
@@ -314,4 +314,57 @@ async def async_update(self) -> None:
         DEVICE_CLASS: SensorDeviceClass.CO2,
         DEVICE_UNITS: CONCENTRATION_PARTS_PER_MILLION,
     },
+    "exhaust_fan_speed": {
```

Comment:
Please import the constants from **ramses_rf** into **const.py** and import into here.